rollup/plugins/bundlePlugins.js

```diff
@@ -119,6 +119,8 @@ export function makeTerserPlugin() {
           // We want to keept he _replay and _isEnabled variable unmangled to enable integration tests to access it
           '_replay',
           '_isEnabled',
+          // We want to keep the _integrations variable unmangled to send all installed integrations from replay
+          '_integrations',
```

Comment:
Can we git rid of this by making this public? But we can also tackle this in a follow up PR!

Reply:
Could be optional on `Client` interface I think then yes.

Reply:
We can but the effect of the bundle size is the same though...

Reply:
let's leave it for now and think about this later!